### DISCUSSION TITLE
域名登录时使用admin账号密码登录提示登录成功但是无跳转

### DISCUSSION THREAD

**Discussion by HelloDanielCui** [AlistGo/alist#6908] [Q&A]
域名登录时使用admin账号密码登录提示登录成功但是无跳转，同时提示Guest user is disabled, login please。页面始终保持这样。
<img width="854" alt="chrome_XGm49dglo5" src="https://github.com/user-attachments/assets/7cd8e94e-332a-4079-b9ec-c1cf06d7301c">

**Comment by Simonzhang8** [comment]
域名登录？意思是IP访问时正常吗

**Reply by HelloDanielCui** [reply]
IP访问也是这样ರ_ರ

**Comment by jjcszxh** [comment]
同样问题，套了cdn存在
